Yauheni_Mazurau/04.GitOps/README.md.txt

```diff
@@ -0,0 +1,71 @@
+Homework 04.GitOps
+

```

Comment:
Please use code section in github markdown
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks